### PR TITLE
snmp.yml: Scrape sysUpTime

### PR DESCRIPTION
**Pull request by RichiH**

### CODE REVIEW

File: snmp.yml [hunk 1]
```diff
@@ -2,9 +2,12 @@
 # Default module, interface stats.
 default:
   walk:
+    - 1.3.6.1.2.1.1
```
Comment (brian-brazil) [inline snmp.yml]: this can be more specific

File: snmp.yml [hunk 2]
```diff
@@ -2,9 +2,12 @@
 # Default module, interface stats.
```
Comment (brian-brazil) [inline snmp.yml]: this needs updating

**Comment by RichiH** [comment]
Updated.

**Comment by brian-brazil** [comment]
Thanks!

**Comment by RichiH** [comment]
And to you :)

**Pull request merged**